src/PoolManager.sol

```diff
@@ -139,6 +140,13 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
         Lock.lock();
     }
 
+    /// @inheritdoc IPoolManager
+    function sync(Currency currency) public returns (uint256 balance) {
```

Comment:
feel like we should be adding the same modifiers as the other functions? like onlyWhenUnlocked

Reply:
I don't think its necessary to call it inside the lock, like don't see any security vulns with it. But do you prefer it for consistency?

Reply:
mm no thats ok im happy with it outside